hs_access_control/models/user.py

```diff
@@ -1088,7 +1088,9 @@ def owned_resources(self):
         """
 
         if not self.user.is_active:
-            raise PermissionDenied("Requesting user is not active")
+            pass
```

Comment:
This change is too deep for a redirection fix and breaks patterns set in the class.  Try looking at the [UserProfile view](UserProfileView).  Catch the error (with a wide net) and redirect there.

Reply:
thanks @sblack-usu I see what you're saying. 
You can probably just ignore below because perhaps we just need to be deleting resources of inactive users anyway.

But a long explanation of what I was thinking here: These @property decorators like `owned_resources` are used not only for the user making http request, but also of course on all of the `user` instances that might exist in a page context. So there are instances where the `PermissionDenied("Requesting user is not active")` isn't really true.

Ex: I can be logged in as an admin and nav to a public resource page whose owner is deactivated. Even though the user making the request is active, the owner of the resource is not active and an irrelevant "Requesting user is not active" message is given because of the PermissionDenied exception. So the user making the request actually does have permission but is incorrectly told that they don't, because the owner is inactive. TLDR